### PR TITLE
kubevirt: add source and remove type from disks table

### PR DESCRIPTION
**Pull request by suomiy**
changing according to our recent UI discussions 

@glekner @matthewcarleton 

btw, shouldn't we keep the name first?

tat

### CODE REVIEW

**Comment by matthewcarleton** [comment]
@suomiy let me look at the order again. This could just be a artifact of the old Create VM Flow where we had an inline edit flow where source needed to be first because it would determine which fields followed when editing/creating a disk.

**Comment by matthewcarleton** [comment]
@suomiy I think we could now align better with the console by putting the name first. We'll still want to keep the source first in the modal b/c it impacts other fields.

**Comment by suomiy** [comment]
fixed, name comes first now

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-tab.tsx [hunk 1]
```diff
@@ -62,12 +62,14 @@ const getStoragesData = (
       pvcsLoading: !isLoaded(pvcs),
     });
 
+    const source = combinedDisk.getSource();
+
     return {
       wizardStorageData,
       // for sorting
       name: combinedDisk.getName(),
+      source: source && source.getValue(),
```
Comment (glekner) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-tab.tsx]: maybe the safe accessing can move to `getSource()` ?

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-tab.tsx]: fixed

**Comment by matthewcarleton** [comment]
> fixed, name comes first now

ok, this means we'll want to ensure it's the same everywhere now right (Name first)?

**Comment by suomiy** [comment]
> @suomiy I think we could now align better with the console by putting the name first. We'll still want to keep the source first in the modal b/c it impacts other fields.

> ok, this means we'll want to ensure it's the same everywhere now right (Name first)?

not sure what you mean by that? The names are first in all tables, but not in the modals.

**Comment by matthewcarleton** [comment]
> > @suomiy I think we could now align better with the console by putting the name first. We'll still want to keep the source first in the modal b/c it impacts other fields.
> 
> > ok, this means we'll want to ensure it's the same everywhere now right (Name first)?
> 
> not sure what you mean by that? The names are first in all tables, but not in the modals.

ok, we have in some of our designs (maybe just CD-ROMs right now) where the source is still first so maybe that's the only one we need to update which I can do. FYI @glekner

**Pull request merged**